src/BloomBrowserUI/bookEdit/js/textOverPicture.ts

```diff
@@ -71,11 +301,20 @@ class TextOverPictureManager {
             mouseX,
             mouseY
         );
-        // I tried to do without this... it didn't work. This causes page changes to get saved and fills
-        // things in for editing.
-        // It causes EditingModel.RethinkPageAndReloadIt() to get run... which eventually causes
-        // makeTextOverPictureBoxDraggableClickableAndResizable to get called by bloomEditing.ts.
-        BloomApi.postThatMightNavigate("common/saveChangesAndRethinkPageEvent");
+
+        const contentElement = wrapperBox.get(0);
+        const bubbleSpec: BubbleSpec = {
+            version: "1.0",
+            style: "speech",
+            tails: [Bubble.makeDefaultTail(contentElement)],
+            level: 1
+        };
+        const bubble = new Bubble(contentElement);
```

Comment:
It would be better to use the Comical method that gets a default bubble spec for a speech bubble.